src/Controller/ShippingExportController.php

```diff
@@ -30,7 +32,7 @@
      *
      * @return RedirectResponse
      */
-    public function exportAllNewShipmentsAction(Request $request)
+    public function exportAllNewShipmentsAction(Request $request): ?RedirectResponse
```

Comment:
Each action in Symfony must return a Response instance. Null must not be allowed as return type.